packages/react/src/constants/device-sizes.constant.ts

```diff
@@ -21,8 +21,8 @@ const sizes = {
     max: 991,
   },
   large: {
-    min: 990,
-    default: 991,
+    min: 992,
```

Comment:
Given the default `large` having bigger implications, we should avoid this change for now.

Reply:
Yes, I did not want to make this change in default sizes. But if you look at the medium max, it is 991px whereas in the large min, it is 990px. There is some overlap. So to avoid the overlap I had made the change.

On searching in the code, it seems the large min is not being used anywhere. So I'll revert here.